scripts/daikon.bashrc

```diff
@@ -58,7 +58,12 @@ if [ -d ${DAIKONDIR}/java ]; then
 fi
 export DAIKON_CLASSPATH
 
-export PATH=${DAIKONSCRIPTS}:${PLUMESCRIPTS}:${JAVA_HOME}/bin:$PATH
+# Only add JAVA_HOME to PATH if JAVA_HOME is not set; see GitHub issue #493
+if [ -z "$JAVA_HOME" ]; then
```

Comment:
This looks backwards to me.  If JAVA_HOME is *not* set, then this adds `/bin` to the path.
I think that `${JAVA_HOME}/bin` should be added to the path only if JAVA_HOME *is* set.

Reply:
Or maybe the change should be Darwin-specific?  (I'm not sure.)

Reply:
You are right — this is not the best solution. There's a few options:
1. Make the change Darwin-specific
2. Instead of prepending `$JAVA_HOME/bin` to the path, we could append it.
3. Check if a `java` executable is already in the user's `$PATH`, only add `$JAVA_HOME/bin` if one isn't
Any thoughts on which one we should choose? I am leaning towards 2.

Reply:
I'm open to any of these, so your proposal of <span>#</span>2 is fine with me.

Reply:
I've updated this PR with the second fix — please let me know if anything else is needed. Thanks!